index.html

```diff
@@ -87,12 +87,41 @@ <h3>Workshop</h3>
                     <img src="./capstone_assets/program_icon_05.png">
                     <h3>Connect</h3>
                 </div>
-                <p>Network with other producers, make new friends, share your love for music with like-minded people!</p>
+                <p>Network with other producers, make new friends, share your love for music with like-minded people!
+                </p>
             </div>
         </div>
 
         <button>Join Music Summit 2022</button>
+    </section>
+
+    <section class="speakers flex">
+        <div class="flex speakersHeader">
+            <h3>Featured Speakers</h3>
+            <hr>
+        </div>
+
+        <div class="speaker flex">
+            <img src="./capstone_assets/MG.webp" alt="Martin Garrix lastfm Photo">
+            <div class="speakerInfo flex">
+                <h2>Martin Garrix</h2>
+                <span>2nd Best DJ in the World</span>
+                <hr>
+                <p>Awarded with two MTV EMAs in 2016, holding the #1 spot in the 2017 DJ Mag Top 100.</p>
+            </div>
+        </div>
+
+        <div class="speaker flex">
+            <img src="./capstone_assets/DG.webp" alt="David Guetta lastfm Photo">
+            <div class="speakerInfo flex">
+                <h2>David Guetta<h2>
+                <span>1st Best DJ in the World</span>
+                <hr>
+                <p>He has over 50 million record sales globally, with more than 10 billion streams.</p>
+            </div>
+        </div>
 
+        <button>MORE <img src="./capstone_assets/dropdown.png"></button>
```

Comment:
- [ ] Great job implementing the functionality of the `more` button 💯 Kindly remove that button from the desktop view and make sure all speakers are being displayed on the desktop view. This is because the button should only be present in the mobile view.
- Your design:
![Annotation 2022-05-25 151728](https://user-images.githubusercontent.com/85113567/170285455-6e778e05-4595-4c17-905b-08be4913c859.jpg)
- Required design:
![Annotation 2022-05-25 151748](https://user-images.githubusercontent.com/85113567/170285490-1bb62d2b-11bc-4ded-b45f-0f6ce10e4a4b.jpg)